lib/fizzy/instantiate.hpp

```diff
@@ -81,20 +81,28 @@ using bytes_ptr = std::unique_ptr<bytes, void (*)(bytes*)>;
 /// The module instance.
 struct Instance
 {
+    /// Module of this instance.
     std::unique_ptr<const Module> module;
-    // Memory is either allocated and owned by the instance or imported as already allocated bytes
-    // and owned externally.
-    // For these cases unique_ptr would either have a normal deleter or noop deleter respectively
+    /// Instance memory.
+    /// Memory is either allocated and owned by the instance or imported as already allocated bytes
+    /// and owned externally.
+    /// For these cases unique_ptr would either have a normal deleter or noop deleter respectively
```

Comment:
```suggestion
    /// For these cases unique_ptr would either have a normal deleter or no-op deleter respectively
```

Reply:
Is it "deleter" or destructor?

Reply:
> Is it "deleter" or destructor?

It refers to `unique_ptr`'s deleter

![image](https://user-images.githubusercontent.com/1863135/106478116-30729b80-64a9-11eb-84dc-f0ed22a913be.png)